backend/app/views/spree/admin/orders/_form.html.erb

```diff
@@ -1,3 +1,5 @@
+<% admin_breadcrumb(plural_resource_name(Spree::Shipment)) %>
```

Comment:
Wouldn't it be better to always have `admin_breadcrumb` at the top of every **non-partial** template?

Like this we ensure that the method is called only once. Otherwise, we can't reuse `_form` partial without getting the "Shipments" breadcrumb, no matter which page we're on.